src/quicklook.cs

```diff
@@ -127,6 +125,7 @@ interface QLPreviewControllerDelegate {
 		[Export ("previewController:shouldOpenURL:forPreviewItem:"), DelegateName ("QLOpenUrl"), DefaultValue (false)]
 		bool ShouldOpenUrl (QLPreviewController controller, NSUrl url, [Protocolize] QLPreviewItem item);
 
+#if !MONOMAC
```

Comment:
This one is already inside a !MONOMAC from line 49, but I understand this is a quick PR!

Reply:
Yep, I decided to revert and make a few changes as humanly possible to get it fixed without losing the define/attribute quality. 

I'll leave refactoring to a future day.